client/cli/src/params/import_params.rs

```diff
@@ -55,6 +56,12 @@ pub struct ImportParams {
 	)]
 	pub wasm_method: WasmExecutionMethod,
 
+	/// Specify the path where local WASM runtimes are stored.
+	///
+	/// These runtimes will override on-chain runtimes when the version matches.
+	#[structopt(long, value_name = "PATH", parse(from_os_str))]
+	pub wasm_runtime_overwrites: Option<PathBuf>,
```

Comment:
Just a terminology query - should we use `override` vs `overwrite` everywhere? Seems like overwrite is used predominantly but I think override is a better choice - either way good to be consistent, unless actually there is some difference in meaning - in which case perhaps choose another word, so that the two concepts are more obviously distinct in their name.

Reply:
I like `override` better too (and may be the one to blame for the inconsistent docs here, not sure) but kept my mouth shut for fear of nitpicking, but as you opened up the can... :) 

"Overwrite" is more concrete than "override" and in some sense we _are_ writing something over what was previously there. We're "clobbering" the on-chain runtime. On the other hand "override" reflects more on what the actual outcome of the operation is: taking control over the normal flow of a process (picking the runtime) and intervene. It's very close to the dictionary definition of "override": "interrupt the action of (an automatic device), typically in order to take manual control.".

Either works. I like "override" better. Andrews call.